src/errors.rs

```diff
@@ -210,6 +210,12 @@ pub enum OrandaError {
     #[diagnostic(help = "You can manually specify path in your components.mdbook config")]
     MdBookConfigInvalid,
 
+    #[error("Specified path `{path}` was not found on your filesystem!")]
+    #[diagnostic(
+        help = "Make sure you're in the same directory where your oranda.json or README file is!"
```

Comment:
we may want this to say "Make sure you specify your path relative to the README/oranda.json" though this is tricky because sometimes the error is due to an inferred path, where other times its a specified path. ideally- those 2 cases have different errors because they have different fixes